src/Client/Client.php

```diff
@@ -51,6 +51,7 @@
     private $accountId;
 
     /**
+     * Client constructor.
```

Comment:
Dat het een constructor is, zien we wel aan de naam van de methode (non blocking)